include/cuda/std/detail/libcxx/include/type_traits

```diff
@@ -4767,6 +4767,8 @@ struct __has_operator_addressof
 
 #endif  // _LIBCUDACXX_CXX03_LANG
 
+template <class...> using _Void_t = void;
```

Comment:
can we just change the `#if _LIBCUDACXX_STD_VER > 11` below as well? Those are already backported from C++17 to C++14. Might as well go to 11 if we can.

Reply:
yeah that sounds like a good idea

Reply:
actually scratch that. It is only used for container like types. And there the issue is that we rely on `cuda::std::data` and `cuda::std::size` which are only c++17.

So we do not even need to change that at all unless we want to also backport those